src/Components/Aspire.Confluent.Kafka/MetricsService.cs

```diff
@@ -0,0 +1,83 @@
+        {
+            while (channel.Reader.TryRead(out var json))
+            {
+                Statistics? statistics;
+                try
+                {
+                    statistics = JsonSerializer.Deserialize(json, StatisticsJsonSerializerContext.Default.Statistics);
+                }
+                catch
+                {
+                    LogDeserializationWarning(json);
+                    continue;
+                }
+
+                if (statistics == null || statistics.Name == null)
+                {
+                    LogDeserializationWarning(json);
+                    continue;
+                }
+
+                TagList tags = new()
+                {
+                    { Tags.ClientId, statistics.ClientId },
+                    { Tags.Name, statistics.Name }
+                };
+
+                metrics.ReplyQueueMeasurements.Enqueue(new Measurement<long>(statistics.ReplyQueue, tags));
+                metrics.MessageCountMeasurements.Enqueue(new Measurement<long>(statistics.MessageCount, tags));
+                metrics.MessageSizeMeasurements.Enqueue(new Measurement<long>(statistics.MessageSize, tags));
+
+                tags.Add(new KeyValuePair<string, object?> (Tags.Type, statistics.Type));
+
+                if (_state.TryGetValue(statistics.Name, out var previous))
+                {
+                    metrics.Tx.Add(statistics.Tx - previous.Tx, tags);
+                    metrics.TxBytes.Add(statistics.TxBytes - previous.TxBytes, tags);
+                    metrics.TxMessages.Add(statistics.TxMessages - previous.TxMessages, tags);
+                    metrics.TxMessageBytes.Add(statistics.TxMessageBytes - previous.TxMessageBytes, tags);
+                    metrics.Rx.Add(statistics.Rx - previous.Rx, tags);
+                    metrics.RxBytes.Add(statistics.RxBytes - previous.RxBytes, tags);
+                    metrics.RxMessages.Add(statistics.RxMessages - previous.RxMessages, tags);
+                    metrics.RxMessageBytes.Add(statistics.RxMessageBytes - previous.RxMessageBytes, tags);
+                }
+                else
+                {
+                    metrics.Tx.Add(statistics.Tx, tags);
+                    metrics.TxBytes.Add(statistics.TxBytes, tags);
+                    metrics.TxMessages.Add(statistics.TxMessages, tags);
+                    metrics.TxMessageBytes.Add(statistics.TxMessageBytes, tags);
+                    metrics.Rx.Add(statistics.Rx, tags);
+                    metrics.RxBytes.Add(statistics.RxBytes, tags);
+                    metrics.RxMessages.Add(statistics.RxMessages, tags);
+                    metrics.RxMessageBytes.Add(statistics.RxMessageBytes, tags);
+                }
+
+                _state[statistics.Name] = statistics;
+            }
+        }
+    }
+
+    [LoggerMessage(LogLevel.Warning, EventId = 42, Message = "Invalid statistics json payload received: `{json}`")]
```

Comment:
Why `42`?

Reply:
which value would you suggest ?

Reply:
Looking at other usages around dotnet/aspnetcore, it looks like the convention is to start at `1`.